assets/js/components/Settings/Settings.jsx

```diff
@@ -0,0 +1,159 @@
+import React, { useState, useEffect } from 'react';
+import { get } from 'axios';
+
+import Button from '@components/Button';
+
+import { logError } from '@lib/log';
+
+import { Transition } from '@headlessui/react';
+import classNames from 'classnames';
+
+const Settings = () => {
+  const [loading, setLoading] = useState(false);
+  const [apiKey, setApiKey] = useState(null);
+  const [showApiKey, setShowApiKey] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+    get('/api/installation/api-key')
+      .then(({ data: { api_key: apiKey } }) => {
+        apiKey !== undefined && setApiKey(apiKey);
+        setLoading(false);
+      })
+      .catch((error) => {
+        logError(error);
+        setLoading(false);
+      });
+  }, []);
+
+  const hasApiKey = apiKey !== null;
```

Comment:
Would eslint complain if we put `Boolean(apiKey)` here?

Reply:
does not complain it seems. Gonna change in one commit with the previous comment.